modmesh/pylibmgr.py

```diff
@@ -33,7 +33,7 @@
 lib_path = {}
 
 
-class mm_path_finder(importlib.abc.MetaPathFinder):
+class MMPathFinder(importlib.abc.MetaPathFinder):
```

Comment:
Do we need `MM` here?

Reply:
Something indicating the meta path class is for modmesh is very helpful in debugging

Reply:
I have changed it from `MM` to `Modmesh`.
`MM` looks weird in camel case style.

Reply:
`Modmesh` looks better.